R/plot_with_settings.R

```diff
@@ -252,7 +282,7 @@ plot_with_settings_srv <- function(id,
         "trel"
       } else if (inherits(plot_r(), "grob")) {
         "grob"
-      } else if (inherits(plot_r(), c("NULL", "histogram")) && !inherits(plot_r, "reactive")) {
+      } else if (inherits(plot_r(), c("NULL", "histogram", "list")) && !inherits(plot_r, "reactive")) {
```

Comment:
even if sb provide function like function () list(). The plot will be empty and the download not work BUT the app will NOT crash.